Linking views to holding

Machinewarlock opened the pull request.

Machinewarlock committed:
Linked

All the views are now linked to the holding page and corrected the
username showing in the 'are you happy question'

fixed issue: Remove duplicate firebase config from
app/onboarding/index.html

Machinewarlock committed:
Fixed username

Fixed username showing in clothing, emotion and want more views

Machinewarlock committed:
Fixed buttons being in nav

Added david-container

aliblackwell commented on app/checkin/checkin.js:
```diff
@@ -29,7 +29,7 @@ function processCheckin() {
       if (distance < 1000) {
 
-        window.location.href = '/app/set-username';//finishedCurrentView();
+      finishedCurrentView();
```
@Machinewarlock checkin should redirect to onboarding manually, and the database will start out with '/app/onboarding/'

Machinewarlock committed:
Fixed

Checkin goes straight to onboarding

Machinewarlock committed:
Added holding to davidtolife

After clicking 'heart' 10 times view changes to holding

Machinewarlock committed:
Added a helper function

Helper function of startCurrentView which tells the app to stay on the
holding page until a new view is selected by the dashboard app.

LeticiaBriscoe committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into Linking-views-to-holding
